KarhooUISDK/CommonUI/MapMVP/AppleMapKit/KarhooMKMapView.swift

```diff
@@ -148,6 +149,22 @@ final class KarhooMKMapView: UIView, MapView, UIGestureRecognizerDelegate {
         zoom(toLevel: standardZoom)
     }
 
+    @discardableResult
+    func zoomToUserPosition() -> Bool {
+        if let userLocation = CLLocationManager().location?.coordinate {
+            let viewRegion = MKCoordinateRegion(
+                center: userLocation,
+                latitudinalMeters: CLLocationDistance(standardZoom),
+                longitudinalMeters: CLLocationDistance(standardZoom)
+            )
+            mapView.setRegion(viewRegion, animated: true)
+            return true
+        } else {
+            assertionFailure()
```

Comment:
This causes the app to crash instead of just returning false. Please remove it

Reply:
The crash'd to occur only on debug setup. Nonetheless, I removed it.